modules/standard/Random.chpl

```diff
@@ -504,15 +583,92 @@ module Random {
 
 
     /*
-      Models a stream of pseudorandom numbers.  See the module-level
-      notes for :mod:`PCGRandom` for details on the PRNG used.
+
+      Models a stream of pseudorandom numbers generated by the PCG random number
+      generator.  See http://www.pcg-random.org/ and the paper, "PCG: A Family
+      of Simple Fast Space-Efficient Statistically Good Algorithms for Random
+      Number Generation" by M.E. O'Neill.
+
+      This class builds upon the :record:`pcg_setseq_64_xsh_rr_32_rng` PCG RNG
+      which has 64 bits of state and 32 bits of output.
+
+      While the PCG RNG used here is believed to have good statistical
+      properties, it is not suitable for cryptographic use. Additionally,
```

Comment:
For a Monte Carlo simulation (for example) we don't care if somebody could predict the next random number based on the last one. But we would care if the RNG was generating key material.
I'm just trying to say it would be a mistake to use PCG or NPB or probably any other RNG we will support to create "random" numbers to create keying material, because these RNGs are predictable. This statement is pretty much true for all RNGs. Things like /dev/random try to incorporate environmental information to create keying material. That's a very different problem that we aren't solving here.